src/components/SearchResults/SearchResults.js

```diff
@@ -103,7 +107,15 @@ class SearchResults extends Component {
         />
       );
     default:
-      return null;
+      return (
+        <SearchResult
+          variant="default"
+          type={typeName}
+          title={item.title || item.name}
+          source={item.source}
+          onClick={() => onResultClick(item)}
```

Comment:
Just to avoid errors when no handler is set, maybe you could add a check?
`onClick={() => onResultClick && onResultClick(item)}`

Reply:
Yes good catch, I will add a fail-safe

Reply:
@royschut updated the PR